packages/dnb-eufemia/src/extensions/payment-card/PaymentCard.js

```diff
@@ -236,6 +237,52 @@ function StatusOverlay({ cardStatus, translations, skeleton }) {
           </div>
         </div>
       )
+
+    case 'order_in_process':
+      return (
+      <div
+        className={classnames(
+          'dnb-payment-card__blocking__overlay',
+          createSkeletonClass('font', skeleton)
+        )}
+      >
+        <div className="dnb-payment-card__blocking__center">
+          <StatusIcon status={cardStatus} />
+          <P top="xx-small">{translations.text_order_in_process}</P>
+        </div>
+      </div>
```

Comment:
```suggestion
      <BlockingOverlay text={translations.text_order_in_process} />
```

Looks like all tree blocks are the same, except the "text". So I suggest to make a shared component above, and consume it in each case.

Reply:
yeah can see that its the same, will change it to make use of an component instead